src/main/java/org/opensearch/security/dlic/rest/api/MigrateApiAction.java

```diff
@@ -115,7 +179,20 @@ protected void migrate(final RestChannel channel, final Client client) throws IO
         final SecurityDynamicConfiguration<?> loadedConfig = load(CType.CONFIG, true);
 
         if (loadedConfig.getVersion() != 1) {
-            badRequest(channel, "Can not migrate configuration because it was already migrated.");
+            final ImmutableList.Builder<SecurityDynamicConfiguration<?>> builder = ImmutableList.builder();
+            for (CType<?> ctype : C_TYPES) {
```

Comment:
nit: can we directly use ``Ctype.values()`` ?

Reply:
Changed this to CType.values().